clojure/defs.bzl

```diff
@@ -1,31 +1,45 @@
+# ; on windows, : on linux...
+_CLASSPATH_SEPARATOR = ";"
```

Comment:
What is this, some sick joke?

Reply:
Indeed

Reply:
Im sure there is a way to select it properly based on platform